src-v2/Judge/NTS.Judge/Events/CompetitionChildrenBehind.cs

```diff
@@ -2,9 +2,10 @@
 using Not.Blazor.Ports.Behinds;
 using Not.Exceptions;
 using NTS.Domain.Setup.Entities;
+using System.Collections.Generic;
 
 namespace NTS.Judge.Events;
-public class CompetitionChildrenBehind : INotBehindParent<Contestant>, INotBehindParent<Phase>, INotBehindWithChildren<Competition>
+public class CompetitionChildrenBehind : INotSetBehind<Contestant>, INotSetBehind<Phase>, INotBehindWithChildren<Competition>
```

Comment:
This might be a good place to rename the odd `INotBehindWithChildren` to a more reasonable `INotBehindParent` since  we no longer have another interface conflicting.

Reply:
Renamed the interface, however used the name `INotParentBehind` to avoid the need to remove the `INotBehindParent` yet. And also comply with the emerging naming convention -> INot-*semantic part of the name*-Behind. 

Looking forward to hearing what you think about that approach.

Reply:
Sounds great, I like consistency. However why not delete the old `INotBehindParent`?

Reply:
Done.